### PR TITLE
Fix wifi on moto g7 power

### PR DESCRIPTION
**Pull request by luka177**

### CODE REVIEW

**Review by M0Rf30 — APPROVED** [review]

File: arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts [hunk 1]
```diff
@@ -457,6 +457,8 @@
 };
 
 &wcnss_iris {
+	compatible = "qcom,wcn3660b"; /* WCN3680B */
```
Comment (z3ntu) [inline arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:460]: Use `qcom,wcn3680` if your device has a WCN3680B

Comment (barni2000) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:460]: Afaik moto ocean has 3660b it was find out by @ultra-azu

Comment (barni2000) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:460]: https://github.com/ultra-azu/linux/commits/alikates/6.5/main

**Comment by barni2000** [comment]
Can you rebase this for 6.6.0/main?

**Comment by M0Rf30** [comment]
Cherry-picked on top of 6.6.10/main

**Pull request closed without merging**